### DISCUSSION TITLE
Footnotes in slides

### DISCUSSION THREAD

**Discussion by klancaster-unr** [Mogztter/asciidoctor-web-pdf#601] [Q&A]
I can create footnotes in slides, however, the text is never displayed - only the number. Is there a way to have the footnote text appear either on the slide where the note was added or at the end of the slide deck?

**Comment by ggrossetie** [comment]
Are you referring to the slides example: https://github.com/Mogztter/asciidoctor-web-pdf/tree/main/examples/slides?
Footnotes are supported through Paged.js. The slides example is using a custom converter (see: https://github.com/Mogztter/asciidoctor-web-pdf/blob/main/examples/slides/template.js) which does not rely on Paged.js to paginate the content. 

As a result, you will need to implement the footnote logic in the custom converter using `Document.getFootnotes()`: https://asciidoctor.github.io/asciidoctor.js/2.2.5/#documentgetfootnotes
